modules/Microsoft.Network/localNetworkGateways/.test/min/dependencies.bicep

```diff
@@ -0,0 +1,14 @@
+@description('Optional. The location to deploy to.')
```

Comment:
Please remove file, not needed for min test.

Reply:
done